src/index.ts

```diff
@@ -63,65 +64,44 @@ const slackWebhookOption = new Option(
   "Slack webhook URL"
 ).env("SLACK_WEBHOOK");
 
-const watchdogTimeoutOption = new Option(
-  "--watchdog-timeout <watchdogTimeout>",
-  "Watchdog timeout (in seconds)"
-)
-  .default("30")
-  .env("WATCHDOG_TIMEOUT")
-  .argParser(parseIntOption);
-
 const databasePathOption = new Option(
   "--database-path <databasePath>",
   "Path to the database"
 )
   .default(DEFAULT_DATABASE_PATH)
   .env("DATABASE_PATH");
 
-const multiRpcOption = new Option(
-  "--rpc <rpc...>",
-  "Chain RPC endpoints to monitor"
-).makeOptionMandatory(true);
-
-const rpcOption = new Option("--rpc <rpc>", "Chain RPC endpoint to monitor")
+const chainConfigHelp = `Chain configuration in the format of <rpc>,<deploymentBlock>[,<watchdogTimeout>,<orderBookApi>], e.g. http://erigon.dappnode:8545,12345678,30,https://api.cow.fi/mainnet`;
+const multiChainConfigOption = new Option(
+  "--chain-config <chainConfig...>",
+  chainConfigHelp
+)
   .makeOptionMandatory(true)
-  .env("RPC");
-
-const multiDeploymentBlockOption = new Option(
-  "--deployment-block <deploymentBlocks...>",
-  "Block number at which the ComposableCoW contract was deployed on the respective chains"
-).makeOptionMandatory(true);
+  .argParser(parseChainConfigOptions);
 
-const deploymentBlockOption = new Option(
-  "--deployment-block <deploymentBlock>",
-  "Block number at which the ComposableCoW was deployed"
+const chainConfigOption = new Option(
+  "--chain-config <chainConfig>",
+  chainConfigHelp
 )
   .makeOptionMandatory(true)
-  .env("DEPLOYMENT_BLOCK")
-  .argParser(parseIntOption);
-
-const multiOrderBookApiOption = new Option(
-  "--orderBookApi <orderBookApi...>",
-  "Orderbook API base URLs (i.e. https://api.cow.fi/mainnet, https://api.cow.fi/xdai, etc.)"
-).default([]);
+  .env("CHAIN_CONFIG")
+  .argParser(parseChainConfigOption);
 
-const orderBookApiOption = new Option(
-  "--orderBookApi <orderBookApi>",
-  "Orderbook API base URL (i.e. https://api.cow.fi/mainnet)"
-).default(undefined);
+const addressOption = new Option(
+  "--address <address...>",
```

Comment:
isn't this too generic name?

Reply:
What about `safeAddress[es]`?

Reply:
In other places we call it `owner`, not sure if its the best name

Reply:
I set this `--only-owner`